stripper/ze_l0v0l_v1_4_csgo2.cfg

```diff
@@ -141,3 +141,31 @@ modify:
 		"OnStartTouch" "lvl4winfixEndRound_CounterTerroristsWin56-1"
 	}
 }
+
+;Fix particles not appearing
+modify:
+{
+	match:
+ 	{
+  		"classname"	"logic_relay"
+		"targetname" "MapSpawn"
+  	}
+   	insert:
+	{
+ 		"OnSpawn" "heal_eff,DestroyImmediately,,6,1"
+   		"OnSpawn" "ammo_eff,DestroyImmediately,,6,1"
+	 	"OnSpawn" "zgravity_eff,DestroyImmediately,,6,1"
+   		"OnSpawn" "zheal_eff,DestroyImmediately,,6,1"
+	 	"OnSpawn" "beam_eff,DestroyImmediately,,6,1"
+   		"OnSpawn" "wind_eff,DestroyImmediately,,6,1"
+	 	"OnSpawn" "poison_eff,DestroyImmediately,,6,1"
+   		"OnSpawn" "firet_eff,DestroyImmediately,,6,1"
+	 	"OnSpawn" "icet_eff,DestroyImmediately,,6,1"
+   		"OnSpawn" "elect_eff,DestroyImmediately,,6,1"
+	 	"OnSpawn" "poisont_eff,DestroyImmediately,,6,1"
+   		"OnSpawn" "gravityt_eff,DestroyImmediately,,6,1"
+	 	"OnSpawn" "eartht_eff,DestroyImmediately,,6,1"
+   		"OnSpawn" "watert_eff,DestroyImmediately,,6,1"
+	 	"OnSpawn" "pusht_eff,DestroyImmediately,,6,1"
+   	}
+}
```

Comment:
This isn't going to solve anything, as you're not starting them again afterwards.

Regardless, fixing the root of the problem would be better. `item_temp`, `item_position`, `lvl4item_temp` & `lvl4item_position` are all triggered with 0 delay in `MapStart()`, spawning the item templates with start active particles within the 3 second period after round start where particles cannot be safely started. Editing the .nut to delay all these relays by an additional 3 seconds should be fine (or do all the relay outputs individually, and skip messing with the .nut altogether).

Reply:
Updated, and is included in the .nut file, however, im questioning whether or not it might crash server.